### ISSUE TITLE
Some trouble with module Magento_Swatches and product attributes

### ISSUE THREAD

**Issue by AlexSeyFomin** [magento/magento2#3370]
I tried to change label of attributes in module Magento_Swatches. For this I changed "color" attribute's lable in admin side. Filter displayed this changes, but module Magento_Swatches didn't have any changes. I was able to fix it only after re-saving product in the admin area. But this way of solving problem is only valid for a particular product, pages of other products don't have any changes.
![image](https://cloud.githubusercontent.com/assets/16902841/12977143/aa59870e-d0d1-11e5-883e-b07993759229.png)

**Comment by katmoon** [comment]
Hi Alexey,

Which label you changed in Admin:
- 'Default label' on Properties tab 
- or the one in Manage Labels tab?

![shot-2016-02-12_11-11-11](https://cloud.githubusercontent.com/assets/1260101/13002737/88dbc7c4-d179-11e5-9e8c-5890e1c8d865.png)
![shot-2016-02-12_11-11-58](https://cloud.githubusercontent.com/assets/1260101/13002738/8c440714-d179-11e5-84fa-3ee48d7db8d2.png)

**Comment by AlexSeyFomin** [comment]
Both of them. I tried to do this manipulation with attribute "size". I changed only Manage Labels, but didn't see any changes on the page. Only re-saving had helped.

**Comment by katmoon** [comment]
Did you clear page cache? It should have been suggested by Magento after changing the label.

**Comment by AlexSeyFomin** [comment]
![image](https://cloud.githubusercontent.com/assets/16902841/13042738/388b110e-d3cb-11e5-8300-6ad1737f0ef2.png)

**Comment by katmoon** [comment]
Hi Alexey,

I cannot reproduce the issue on the freshest version. It looked like a problem with cache at first glance. I can suggest you do `bin/magento indexer:reindex` and `bin/magento cache:clean` and check if any of these helps.

Thanks,
Katia.

**Comment by vzabaznov** [comment]
@AlexSeyFomin as @katmoon said bug cannot be reproduced on the fresh magento version, if you feel that bug is not fixed please leave us a message or reopen new issue, and provide hash of the commit you had that bug.
